ovn_k8s/common/kubernetes.py

```diff
@@ -17,22 +17,60 @@
 
 import ovs.vlog
 
+from os import path
 from ovn_k8s.common import exceptions
+from ovn_k8s.common.util import ovs_vsctl
 
+CA_CERTIFICATE = "/etc/openvswitch/k8s-ca.crt"
 vlog = ovs.vlog.Vlog("kubernetes")
 
 
+def _get_api_params():
+    ca_certificate = None
+    api_token = None
+    k8s_api_server = ovs_vsctl("--if-exists", "get", "Open_vSwitch", ".",
+                               "external_ids:k8s-api-server").strip('"')
+    if k8s_api_server.startswith("https://"):
+        if not path.isfile(CA_CERTIFICATE):
+            k8s_ca_crt = ovs_vsctl("--if-exists", "get", "Open_vSwitch",
```

Comment:
do you mind adding a log statement with info level stating that you're going to look for a certificate in ovsdb?